package.json

```diff
@@ -7,11 +7,12 @@
     "lint": "eslint \"src/**/*.js\" \"test/**/*.js\" \"examples/**/*.js\"",
     "doc": "jsdoc src/Core/View.js src/Core/Prefab/GlobeView.js src/Core/Layer/Layer.js src/Renderer/ColorLayersOrdering.js src/Renderer/ThreeExtended/GlobeControls.js src/Core/Geographic/Coordinates.js",
     "doclint": "npm run doc -- -t templates/silent",
-    "test": "npm run lint && npm run build && npm run test-examples",
+    "test": "npm run lint && npm run transpile && npm run test-examples",
```

Comment:
We may want to modify `.travis.yml` to run `npm run build` in addition to `npm run test` then.

I'd rather keep the `npm run build` here, and add `npm run transpile` to the list (either here or within the `test-examples` script instead)